corehq/apps/userreports/models.py

```diff
@@ -707,35 +708,32 @@ class AsyncIndicator(models.Model):
     These indicators will be picked up by a queue and placed into celery to be
     saved. Once saved to the data sources, this record will be deleted
     """
-    doc_id = models.CharField(max_length=255, null=False, db_index=True)
+    doc_id = models.CharField(max_length=255, null=False, db_index=True, unique=True)
     doc_type = models.CharField(max_length=126, null=False)
     domain = models.CharField(max_length=126, null=False)
-    pillow = models.CharField(max_length=126, null=False)
     indicator_config_ids = ArrayField(
         models.CharField(max_length=126, null=True, blank=True),
         null=False
     )
     date_created = models.DateTimeField(auto_now_add=True, db_index=True)
+    date_queued = models.DateTimeField(null=True, db_index=True)
 
     class Meta(object):
-        unique_together = ('doc_id', 'pillow',)
```

Comment:
Yeah. The downside is that it makes it harder to reason about which data sources are slow, but i'll probably just randomize the order they are processed so that there are some logs on the slowness